client/src/components/tabs/tabs.jsx

```diff
@@ -0,0 +1,257 @@
+import React, { useState } from 'react';
+import './tabs.scss';
+import PropTypes from 'prop-types';
+import { Button } from '../button/Button/Button';
+import { Dropdown } from '../form/Dropdown/Dropdown';
+
+const Tabs = ({ tabs }) => {
+  return (
+    <div>
+      <div className="Tabs-desktop">
+        <TabsDesktop tabs={tabs} />
+      </div>
+      <div className="Tabs-mobile">
+        <TabsMobile tabs={tabs} />
+      </div>
+    </div>
+  );
+};
+
+const TabsDesktop = ({ tabs }) => {
+  // const location = useLocation();
+  const [currentTab, setCurrentTab] = useState(tabs[0]);
+  return (
+    <div className="tab-main">
+      <div className="tab-container tabs-center">
+        <div className="tabs-center relative">
+          {tabs.map((tab, index) => {
+            {
+              var css_class = tab == currentTab ? 'nav-btn active_link' : 'nav-btn non_active_link';
+            }
+            return (
+              <Button
```

Comment:
You can use the `<ButtonSelector>` component that was recently made to handle the selection and styles for you